backend/fichier/fichier.go

```diff
@@ -44,6 +44,11 @@ func init() {
 			Name:     "shared_folder",
 			Required: false,
 			Advanced: true,
+		}, {
+			Help:     "If you want to download a shared file that is password protected, add this parameter",
+			Name:     "shared_password",
+			Required: false,
+			Advanced: true,
```

Comment:
This should probably say `IsPassword: true,` here - we are entering a password aren't we. This will mean it is stored obfuscated in the config and will need to be revealed.